openedx/core/djangoapps/oauth_dispatch/dot_overrides/views.py

```diff
@@ -3,6 +3,7 @@
 """
 from __future__ import unicode_literals
 
+from __future__ import absolute_import
```

Comment:
Could you please group both import from `__future__` together? Unfortunately this is sometimes a side effect of modernize.